nautobot/extras/tests/test_tags.py

```diff
@@ -110,6 +110,7 @@ def test_create_invalid_tagged_item(self):
             "slug": "test-location",
             "status": Status.objects.get_for_model(Location).first().pk,
             "tags": [tag.id],
+            "location_type": self.location_type.pk,
```

Comment:
interesting!! Might be a bug that we need to check out?

Reply:
Oh, because it's a negative test - it's an invalid request because `tag.id` above isn't a tag that's valid for a `Location` anyway. I'll bet if you take a closer look at the response under the base branch, it's probably complaining about the missing `location_type` *and* complaining about the bad `tags`.